ui/src/app/utils/analytics.ts

```diff
@@ -0,0 +1,22 @@
+declare let gtag: Function;
+
+/**
+ * Triggers a Google Analytics event.
+ *
+ * For guidance on how to use action names, categories, and labels effectively,
+ * see https://support.google.com/analytics/answer/1033068#Anatomy.
+ */
+export function triggerEvent(
+  category: string, action: string, label?: string, value?: number,
+  nonInteraction: Boolean = false) {
```

Comment:
Just curious, for what kind of event would you set `nonInteraction` flag to `true`?

Reply:
Good question – I'd thought it might be useful for some cases, but after re-reading the docs, the only functional impact it would have is on the bounce rate of certain pages (a pageview with a non-interaction event can be considered a bounce, but a pageview with an interaction event will not be considered a bounce). So I've removed it, since it won't likely be needed in the immediate future.

Reply:
Makes sense. Thanks!